lib/native/source/opensmt/opensmt.i

```diff
@@ -0,0 +1,1419 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+%module OsmtNative
```

Comment:
Where does this file come from? Was it auto-generated? Over 1000 lines of code without a clear desciption is quite hard to maintain in the future.

Reply:
It's semi-autogenerated :). I ran doxygen an the OpenSMT headers, then copied the method definitions from there and ran them through sed to get the 'ignore' statements. The whole process was quite messy and it required some extra editing. It might be possible to shrink this file significantly by not listing all the methods, however, SWIG will then generate stub classes for a lot of the other internal classes of OpenSMT, which is also messy. I'll see if I can find a better way, but it may take some time.

Reply:
I read through the SWIG-file and it looks quite easy to understand.
I like the idea of only providing a subset of the API via SWIG, i.e., only the required parts.
Keep it as is.